lib/awful/tag.lua

```diff
@@ -873,6 +874,14 @@ function tag.mt:__call(...)
     return tag.new(...)
 end
 
+-- Extend the luaobject
+object.properties(capi.tag, {
+    getter_class    = tag,
+    setter_class    = tag,
+    getter_fallback = tag.getproperty,
+    setter          = tag.setproperty,
```

Comment:
As @blueyed  pointed out, `awful.tag` have it's own property logic and this should be removed. Once I will have converted that to the new format, it will be possible to use the more simple:

```
 +object.properties(capi.tag, {
+    getter_class    = tag,
+    setter_class    = tag,
+    auto_emit       = true,
})
```

options, but as it is currently written, it would break stuffs, so I will do this in its own PR.